cirq/ops/common_gates_test.py

```diff
@@ -818,3 +818,32 @@ def test_rxyz_circuit_diagram():
         ), """
 q: ───Rz(π)───Rz(-π)───Rz(-π)───Rz(0.5π)───Rz(0.5π)───
     """)
+
+
+def test_trace_distance():
+    foo = sympy.Symbol('foo')
+    sx = cirq.X**foo
+    sy = cirq.Y**foo
+    sz = cirq.Z**foo
+    sh = cirq.H**foo
+    scx = cirq.CX**foo
+    scz = cirq.CZ**foo
+    sswap = cirq.SWAP**foo
+    siswap = cirq.ISWAP**foo
+    assert cirq.trace_distance_bound(sx) == 1
+    assert cirq.trace_distance_bound(sy) == 1
+    assert cirq.trace_distance_bound(sz) == 1
+    assert cirq.trace_distance_bound(scx) == 1
+    assert cirq.trace_distance_bound(scz) == 1
+    assert cirq.trace_distance_bound(sswap) == 1
+    assert cirq.trace_distance_bound(siswap) == 1
+    assert cirq.trace_distance_bound(sh) == 1
+    assert cirq.trace_distance_bound(cirq.X) == 1
+    assert cirq.trace_distance_bound(cirq.Y**-1) == 1
+    assert cirq.trace_distance_bound(cirq.Z**0.5) == np.sin(np.pi / 4)
+    assert cirq.trace_distance_bound(cirq.H**0.25) == np.sin(np.pi / 8)
+    assert cirq.trace_distance_bound(cirq.CX**2) < 1.e-15
+    assert cirq.trace_distance_bound(cirq.CZ**(1 / 9)) == np.sin(np.pi / 18)
+    assert cirq.trace_distance_bound(cirq.SWAP**0.3) == np.sin(3 * np.pi / 20)
```

Comment:
This is brittle.  Use approximate equality.

`3 * ... / 20` is confusing.  Use `0.3 * ... / 2`.

Reply:
Done